integration/airflow/tests/integration/test_integration.py

```diff
@@ -518,7 +518,7 @@ def test_airflow_mapped_task_facet(airflow_db_conn):
     started_events = [event for event in mapped_events if event["eventType"] == "START"]
     assert len(started_events) == 3
     assert started_events[0]["job"]["name"] == f"{dag_id}.{task_id}"
-    assert set([event["job"]["facets"]["sourceCode"]["source"] for event in started_events]) == set(
+    assert set([event["job"]["facets"]["sourceCode"]["sourceCode"] for event in started_events]) == set(
```

Comment:
Should we mention all the changes of facet keys and attributes in a changelog or somewhere else? Just thinking out loud what would be best - maybe it's more important for the OL airflow provider, but still should be mentioned somewhere.

Reply:
In this case it's different. This is something that is fixed and now is correct with JSON Schema spec. Besides, this is OL standalone package, I wouldn't go strongly with deprecations/notices here.